kodilla-spring/src/main/java/com/kodilla/spring/library/LibraryDbController.java

```diff
@@ -2,7 +2,7 @@
 
 import org.springframework.stereotype.Repository;
 
-@Repository
+//@Repository
```

Comment:
Nie wrzucamy do repozytorium zakomentowanego kodu. Proszę, skasuj to :-)

Reply:
Zmiany "zabrały" się z zadaniem. Poprawione.